contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -524,12 +572,46 @@ NXmpes(NXobject):
             kinetic: 
               doc: |
                 Calibrated kinetic energy axis.
-            binding: 
-              doc: |
-                Calibrated binding energy axis.
+          binding:
+            doc: |
+              Calibrated binding energy axis.
+      kx(NX_NUMBER):
+        exists: optional
+        unit: NX_ANY
+        doc: |
+          Calibrated x axis in k-space.
+          Units are 1/Angström.
+      ky(NX_NUMBER):
+        exists: optional
+        unit: NX_ANY
+        doc: |
+          Calibrated y axis in k-space.
+          Units are 1/Angström
+      kz(NX_NUMBER):
+        exists: optional
+        unit: NX_ANY
+        doc: |
+          Calibrated k axis in k-space.
+          Units are 1/Angström.
+      delay(NX_NUMBER):
+        exists: optional
+        unit: NX_TIME
+        doc: |
+          Calibrated delay time.
+      polarization:
+        exists: optional
+        unit: NX_ANY
+        doc: |
+          Polarization of the incoming beam.
+          
+          Can be any of linear polarization angle (degrees), ellipticity (arb. units).
+          Could be a link to /entry/instrument/beam/incident_polarization or
```

Comment:
Can we really directly link this? afaik this is a stokes vector in these fields which wouldn't translate well to a 1D axis. But it got me thinking: How can we deal with cases where an axis value might be more complex than just a single number?

Reply:
Also would one may collect some basic polarization settings like left-circular or right-circular

Reply:
Since @Tommaso-Pincelli gave two options of what he meant by polarization, we could also split this field into two:
```yaml
polarization_angle(NX_FLOAT):
  exists: optional
  unit: NX_ANGLE
  doc: Linear polarization angle of the incoming beam.
ellipticity(NX_FLOAT):
  exists: optional
  unit: NX_ANY
  doc: Ellipticity of the incoming beam.
```
But I agree, this doesn't help with translation from a more complex vector (in this case, the stokes vector of polarization) to a 1D axis.

Reply:
Yes, actually using an additional ellipticity (we should also make this of unit `NX_ANGLE`) would be enough here. This is calculable from the *normalised* Stokes vector (see https://www.thorlabs.com/newgrouppage9.cfm?objectgroup_id=14200). But maybe it's better if we also replace the Stokes vector with `polarization_angle` and `ellipticity` in NXbeam (or add both?)

This would solve the problem that we cannot specify the unit category in NXbeam/polarization properly and we don't collect the intensity anymore, which we previously did with the Stokes vector. So in my view this would also be a "cleaner" representation of polarization state. However, Stokes vectors can be directly measured which might be the greater argument for keeping it. @rettigl and @Tommaso-Pincelli What are your opinions on this?

Reply:
Typically, one plots these things just as function of the actually scanned parameter, i.e. the wave plate angle. Otherwise, one stokes parameter could be retrieved and used